config/toml.go

```diff
@@ -596,6 +596,19 @@ initial_block_retain_height = {{ .Storage.Pruning.DataCompanion.InitialBlockReta
 # already set a block results retain height, this is ignored.
 initial_block_results_retain_height = {{ .Storage.Pruning.DataCompanion.InitialBlockResultsRetainHeight }}
 
+# If set to true, CometBFT will force compaction to happen for databases that support this feature.
```

Comment:
this needs to be removed I think

Reply:
Agree, not part of this PR, in any case.